### DISCUSSION TITLE
【灌水】绳网新人，请问在哪里能接到委托呢？

### DISCUSSION THREAD

**Discussion by luo-luo-o** [share121/inter-knot#1354] [其他]
新人真诚求问，第一次登上大名鼎鼎的绳网，我也可以接委托嘛？接委托有什么要求嘛？

**Comment by lllliufeng** [comment]
这里



> [!WARNING]
> 检测到不明消息，请谨慎甄别（doge）



<details>
<summary>我要接单！</summary>
您有一份新的委托！
<details>
<summary>查看</summary>
接取吗？
<details>
<summary>确认</summary>
正在为您接取中，请稍候...
<details>
<summary>查看结果</summary>
啊，被人抢先了。非常抱歉，我们将为您更换委托。
<details>
<summary>我还要接单！</summary>
查询中，请稍候...
<details>
<summary>查看我的委托</summary>

检测到特别委托《NGGYU》

委托人：流流流流风

是否接取？

<details>
<summary>接取</summary>
正在为您接取中，请稍候...
<details>
<summary>确认</summary>
即将进入委托...
<details>
<summary>开始</summary>
委托开始中...
<details>
<summary>我来了！！！</summary>

### 你被骗了
 ....................„-~~'''''''~~--„„_
..............„-~''-,::::::::::::::::::: ''-„
..........,~''::::::::',:::::::::::::::: ::::|',
.....::::::,-~'''¯¯¯''''~~--~'''¯'''-,:|
.........'|:::::|: : : : : : : : : : : ::: : |,'
........|:::::|: : :-~~---: : : -----: |
.......(¯''~-': : : :'¯°: ',: :|: :°-: :|
.....'....''~-,|: : : : : : ~---': : : :,'
...............|,: : : : : :-~~--: : ::/
......,-''\':\: :'~„„_: : : : : _,-'
__„-';;;;;\:''-,: : : :'~---~''/|
;;;;;/;;;;;;;\: :\: : :____/: :',__
;;;;;;;;;;;;;;',. .''-,:|:::::::|. . |;;;;''-„__
;;;;;;,;;;;;;;;;\. . .''|::::::::|. .,';;;;;;;;;;''-„
;;;;;;;|;;;;;;;;;;;\. . .\:::::,'. ./|;;;;;;;;;;;;|
;;;;;;;\;;;;;;;;;;;',: : :|¯¯|. . .|;;;;;;;;;,';;|
;;;;;;;;;',;;;;;;;;;;;\. . |:::|. . .'',;;;;;;;;|;;/
;;;;;;;;;;\;;;;;;;;;;;\. .|:::|. . . |;;;;;;;;|/
;;;;;;;;;;;;,;;;;;;;;;;|. .\:/. . . .|;;;;;;;;|

委托结束，感谢您的配合。（doge）

</details>
</details>
</details>
</details>
</details>
</details>
</details>
</details>
</details>
</details>

**Reply by lllliufeng** [reply]
感觉光靠展开就可以玩文字冒险游戏了（

**Reply by XxnjiIAJ** [reply]
可恶啊，管不住手
